packages/kit/src/index.ts

```diff
@@ -18,7 +18,7 @@ export type { ExtendConfigOptions, ExtendViteConfigOptions, ExtendWebpackConfigO
 export { assertNuxtCompatibility, checkNuxtCompatibility, getNuxtVersion, hasNuxtCompatibility, isNuxtMajorVersion, normalizeSemanticVersion, isNuxt2, isNuxt3 } from './compatibility'
 export { addComponent, addComponentsDir } from './components'
 export type { AddComponentOptions } from './components'
-export { nuxtCtx, tryUseNuxt, useNuxt } from './context'
+export { getNuxtCtx as nuxtCtx, tryUseNuxt, useNuxt, globalNuxtCtx } from './context'
```

Comment:
we should probably not change the type of `nuxtCtx` to a function. Rather, we should output `globalNuxtCtx` as `nuxtCtx` so we're backwards compatible.